--- xchat-2.8.8/src/common/xchat.h~	2009-08-16 11:40:16.000000000 +0200
+++ xchat-2.8.8/src/common/xchat.h	2013-06-11 13:44:45.861175558 +0200
@@ -1,10 +1,6 @@
 #include "../../config.h"
 
-#include <glib/gslist.h>
-#include <glib/glist.h>
-#include <glib/gutils.h>
-#include <glib/giochannel.h>
-#include <glib/gstrfuncs.h>
+#include <glib.h>
 #include <time.h>			/* need time_t */
 
 #ifndef XCHAT_H
