--- xchat-1.8.7/src/fe-gtk/maingui.c.fix-USE_GNOME	Sun Oct  7 12:21:44 2001
+++ xchat-1.8.7/src/fe-gtk/maingui.c	Sun Oct  7 12:22:05 2001
@@ -937,7 +937,7 @@
 void
 userlist_hide (GtkWidget *igad, session *sess)
 {
-#ifdef USE_GNOMEE
+#ifdef USE_GNOME
 	if (sess->userlisthidden > 0)
 	{
 		if (sess->gui->paned)
@@ -1778,7 +1778,7 @@
 
 	add_tip (sess->gui->topicgad, _("The channel topic"));
 
-#ifdef USE_GNOMEE
+#ifdef USE_GNOME
 	wid = gtkutil_button (sess->gui->window, GNOME_STOCK_PIXMAP_FORWARD, 0,
 								 userlist_hide, sess, 0);
 #else
