--- xchat-2.0.10/xchat.desktop~	Fri Jul 23 11:18:07 2004
+++ xchat-2.0.10/xchat.desktop	Fri Jul 23 11:18:07 2004
@@ -6,15 +6,18 @@
 Comment[fi]=IRC-sovellus
 Comment[fr]=Client IRC
 Comment[lt]=IRC klientas
-Comment[no]=IRC-klient
+Comment[nb]=IRC-klient
+Comment[pl]=Klient IRC
 Comment[pt_BR]=Cliente de IRC
+Comment[ro]=Client de IRC
 Comment[sl]=Odjemalec IRC
 Comment[sv]=IRC-klient
-Comment[ro]=Client de IRC
 Comment[zh_TW]=X-Chat 聊天程式
 Comment=IRC Client
 Exec=xchat
 Icon=xchat.png
 Terminal=false
 Type=Application
-Categories=Application;Network;
+Categories=GTK;Network;IRCClient;
+# vi: encoding=utf-8
+
