--- xchat-2.4.0/configure.in.orig	2004-08-14 09:34:11.000000000 +0200
+++ xchat-2.4.0/configure.in	2004-08-22 11:59:48.620724280 +0200
@@ -10,7 +10,7 @@
 AM_DISABLE_STATIC
 AM_PROG_LIBTOOL
 
-ALL_LINGUAS="am az bg ca cs de el en_GB es et eu fi fr it ja ko lt lv mk ms nl no pt ru sk sl sr sv uk wa zh_CN"
+ALL_LINGUAS="am az bg ca cs de el en_GB es et eu fi fr it ja ko lt lv mk ms nb nl pl pt ru sk sl sr sv uk wa zh_CN"
 AM_GNU_GETTEXT
 
 dnl displaced from acconfig.h
