--- xchat-2.8.8/xchat.desktop~	2018-09-20 18:51:46.000000000 +0200
+++ xchat-2.8.8/xchat.desktop	2018-09-20 18:52:51.804071052 +0200
@@ -8,17 +8,19 @@ Comment[fi]=IRC-sovellus
 Comment[fr]=Client IRC
 Comment[hu]=IRC-kliens
 Comment[lt]=IRC klientas
+Comment[nb]=IRC-klient
 Comment[no]=IRC-klient
+Comment[ro]=Client de IRC
+Comment[pl]=Klient IRC
 Comment[pt_BR]=Cliente de IRC
 Comment[sl]=Odjemalec IRC
 Comment[sv]=IRC-klient
-Comment[ro]=Client de IRC
 Comment[zh_TW]=X-Chat 聊天程式
 Comment=Chat with other people using Internet Relay Chat
 Exec=sh -c "xchat --existing --url %U || exec xchat"
 Icon=xchat
 Terminal=false
 Type=Application
-Categories=Network;
+Categories=GTK;Network;IRCClient;
 StartupNotify=true
 MimeType=x-scheme-handler/irc;x-scheme-handler/ircs;
