--- xchat-2.4.4/configure.in~	2005-08-28 21:48:43.131905736 +0200
+++ xchat-2.4.4/configure.in	2005-08-28 21:49:12.279474632 +0200
@@ -635,13 +635,13 @@
 PLUGIN_INCLUDES='-I$(top_srcdir)/plugins'
 AC_SUBST(PLUGIN_INCLUDES)
 
+AC_DEFINE_UNQUOTED(XCHATLIBDIR, "${libdir}/xchat")
+
 if test "x${prefix}" = "xNONE"; then
 	AC_DEFINE_UNQUOTED(PREFIX, "${ac_default_prefix}")
-	AC_DEFINE_UNQUOTED(XCHATLIBDIR, "${ac_default_prefix}/lib/xchat")
 	AC_DEFINE_UNQUOTED(XCHATSHAREDIR, "${ac_default_prefix}/share/xchat")
 else
 	AC_DEFINE_UNQUOTED(PREFIX, "${prefix}")
-	AC_DEFINE_UNQUOTED(XCHATLIBDIR, "${prefix}/lib/xchat")
 	AC_DEFINE_UNQUOTED(XCHATSHAREDIR, "${prefix}/share/xchat")
 fi
 
