--- xchat-1.8.8/src/common/xchat.c	Tue Feb 12 03:39:15 2002
+++ xchat-1.8.8/src/common/xchat.orig	Fri Mar 15 19:00:40 2002
@@ -112,8 +112,8 @@
 
 	fe_add_rawlog (serv, tbuf, TRUE);
 
-	if (!EMIT_SIGNAL (XP_IF_SEND, GINT_TO_POINTER (serv->sok), tbuf,
-							GINT_TO_POINTER (len), NULL, NULL, 0))
+	if (!EMIT_SIGNAL (XP_IF_SEND, (void *) serv->sok, tbuf, (void *)len,
+							NULL, NULL, 0))
 	{
 #ifdef USE_OPENSSL
 		if (!serv->ssl)
@@ -912,9 +912,7 @@
 	"NAME Clear\n"				"CMD /clear\n\n")
 
 #define defaultconf_replace \
-	"NAME teh\n"				"CMD the\n\n"\
-	"NAME r\n"					"CMD are\n\n"\
-	"NAME u\n"					"CMD you\n\n"
+             " \n"		" \n"\
 
 #define defaultconf_commands \
    "NAME ACTION\n"		"CMD /me &2\n\n"\
@@ -974,6 +972,7 @@
 
 #ifdef WIN32
 #define defaultconf_urlhandlers \
+	"NAME Open URL\n"							"CMD !start %s\n\n"\
 	"NAME Connect as IRC server\n"		"CMD /newserver %s\n\n"
 #else
 #define defaultconf_urlhandlers _( \
@@ -1106,7 +1105,7 @@
 	if (pid != -1)
 	/* zombie avoiding system. Don't ask! it has to be like this to work
       with zvt (which overrides the default handler) */
-		fe_timeout_add (1000, child_handler, GINT_TO_POINTER (pid));
+		fe_timeout_add (1000, child_handler, (void *)pid);
 #endif
 }
 
